packages/testcontainers/src/containers/DeFiDContainer.ts

```diff
@@ -65,7 +66,7 @@ export abstract class DeFiDContainer extends DockerContainer {
    */
   protected getCmd (opts: StartOptions): string[] {
     return [
-      'defid',
+      'bin/defid',
```

Comment:
We can remove this as in new working build defid path will get set as expected
```suggestion
      'defid',
```